core/kazoo_number_manager/src/kazoo_number_manager_maintenance.erl

```diff
@@ -82,10 +82,11 @@ refresh_numbers_db(Suffix) ->
 -spec fix_accounts_numbers([ne_binary()]) -> 'ok'.
 -spec fix_account_numbers(ne_binary()) -> 'ok'.
 fix_accounts_numbers(Accounts) ->
+    _ = purge_discovery(),
```

Comment:
Just because it reduces the amount of things migration will run through, thus making it run faster.
I don't think clearing that cache is so bad: searches are ran much more often than migrations, no?